README.md

```diff
@@ -3,60 +3,70 @@
 
 [Helm](https://helm.sh) must be installed to use the charts.  Please refer to
 Helm's [documentation](https://helm.sh/docs) to get started.
 
 Once Helm has been set up correctly, add the repo as follows:
 
-  helm repo add galasa https://galasa-dev.github.io/helm
+```
+helm repo add galasa https://galasa-dev.github.io/helm
+```
 
 If you had already added this repo earlier, run `helm repo update` to retrieve
-the latest versions of the packages.  You can then run `helm search repo galasa` to see the charts.
+the latest versions of the packages. You can then run `helm search repo galasa` to see the charts.
 
+If you would like to install the chart into minikube, ensure you have minikube [installed](https://minikube.sigs.k8s.io/docs/start/) and that it is running with `minikube status`. If minikube is not running, start it by running `minikube start`. 
 
 ## Galasa Ecosystem chart
+### RBAC
+If RBAC is active on your Kubernetes cluster, you will need to get your Kubernetes administrator to modify and apply an RBAC yaml file [(rbac-admin.yaml)](./charts/ecosystem/rbac-admin.yaml) to allow authorised users to run the helm install/upgrade/delete commands to interact with the helm chart.
 
-If RBAC is active on your Kubernetes cluster, you will need to get your Kubernetes administrator to apply a couple of RBAC yaml files to allow the Galasa Ecosystem to operate and for you to install the helm chart.
-
-The first RBAC file which can be found at [here](https://raw.githubusercontent.com/galasa-dev/helm/release/charts/ecosystem/rbac.yaml) can be applied without modification.   It allows Galasa to complete the installation, creates a Galasa service account so the Engine Controller can create and manage engine pods.
+The other RBAC file [(rbac.yaml)](./charts/ecosystem/templates/rbac.yaml) is applied automatically when installing the ecosystem chart. It creates a Galasa service account if one does not already exist so the API, Engine Controller, Metrics, and Resource Monitor can coordinate, while allowing the Engine Controller to create and manage engine pods.
 
-The second RBAC file which can be found at [here](https://raw.githubusercontent.com/galasa-dev/helm/release/charts/ecosystem/rbac-admin.yaml) allows someone to do the helm install/upgrade/delete commands, this will need to be modified slightly to include authorised users.
 
-To install a Galasa Ecosystem using Helm, use the following command (after adding the repo detailed above) :-
+### Installation
+To install a Galasa Ecosystem using Helm, use the following command (after adding the repo detailed above). Note: The Galasa Ecosystem Helm chart will deploy three persistent volumes. If you need to provide a Kubernetes storage class for these PVs, you can override the `storageClass` value as shown in the command below. If you are deploying to minikube, you can use the `standard` storage class created for you by minikube, but this is not required.
```

Comment:
I see the helm repo add galasa stuff as being part of the install, not a pre-step. As it's galasa-specific..

Reply:
> If you need to provide a Kubernetes storage class for these PVs, you can override the `storageClass` value as shown in the command below.

I would have 'review your settings.yaml file. Specifically check: ... and then detail what each does. Hopefully the yaml file has enough comments inside to explain the values and what they are for, and how they could be tailored ?

Reply:
Probably best to put "If you are deploying to minikube, you can use the `standard` storage class created for you by minikube, but this is not required." into a call-out ?
rather than in same paragraph ?

Reply:
Addressed in https://github.com/galasa-dev/helm/pull/1/commits/fad3b94c4581df0dae003f50aac19569f88aa2f1